flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches":["x86_64"]
```

Comment:
Why is this x86_64? It's a JavaScript application and I didn't see an explanation. (I could have missed it)

Reply:
Currently we only build for x86_64. We don't have plans to build for any other archs any time soon. Hence we would like to keep all packages relatively the same